source/includes/fact-short-sync-job-desc.rst

```diff
@@ -0,0 +1,3 @@
+- Snapshot which migrates all the data and then stops.
```

Comment:
[Suggestion]

I really liked the way this read in your SQL Server page but that is also just a personal preference.

- Snapshot sync jobs migrate all the data and then stop.
- Continuous sync job run a snapshot and then enter a CDC stage to continuously replicate data changes.

Reply:
I will make this an include to preserve the definitions on each page